credentials/src/main/kotlin/web5/sdk/credentials/StatusListCredential.kt

```diff
@@ -220,18 +223,26 @@ public object StatusListCredential {
   }
 
   private suspend fun HttpClient.fetchStatusListCredential(url: String): VerifiableCredential {
+    val response: HttpResponse
+
+    try {
+      response = this.get(url)
+    } catch (e: Exception) {
+      throw RuntimeException("Failed to retrieve VerifiableCredentialType from $url", e)
+    }
+
     try {
-      val response: io.ktor.client.statement.HttpResponse = this.get(url)
       if (response.status.isSuccess()) {
         val body = response.bodyAsText()
         return VerifiableCredential.parseJwt(body)
       } else {
-        throw ClientRequestException(response, "Failed to retrieve VerifiableCredentialType from $url")
+        throw ClientRequestException(
```

Comment:
Throwing inside a `try` block seems like a code smell to me. Is that intentional?